leukeleu_django_checks/checks.py

```diff
@@ -112,3 +117,16 @@ def check_sentry_dsn(app_configs, **kwargs):
         return [W006]
     else:
         return []
+
+
+# Register checks
+#
+# Do not use the @register decorator as it is typed as Any in
+# django-stubs. This results in mypy errors about the checks
+# becoming untyped when using the decorator.
+# https://github.com/typeddjango/django-stubs/issues/1098
```

Comment:
In case someone picks up this PR; this issue has been fixed, can use decorators again.